src/api/app/controllers/application_controller.rb

```diff
@@ -185,11 +250,8 @@ def extract_user
     if mode == :on
       extract_proxy_user
     else
-      @auth_method = :basic
-
-      extract_basic_auth_user
-
-      @http_user = User.find_with_credentials @login, @passwd if @login
+      extract_auth_user
+      @http_user = User.find_with_credentials @login, @passwd if @login && @passwd
```

Comment:
Enabling Kerberos based authentication doesn't disable password based authentication. If username and password were both provided they are used.